package.json

```diff
@@ -88,6 +88,7 @@
     "@fluentui/react-icons": "^2.0.154-beta.5",
     "@griffel/babel-preset": "1.0.0",
     "@griffel/jest-serializer": "1.0.0",
+    "@griffel/react": "1.0.0",
```

Comment:
We have `import/no-extraneous-dependencies` lint rule that ensures:
- all `dependencies` are declared in package's `package.json`
- all `devDependencies` are declared in package's `package.json` or in the root one

![image](https://user-images.githubusercontent.com/14183168/150830737-62dbe1dd-cfda-4499-ba73-c5660d6f8ba9.png)

As it warns for stories and it's `devDependency`, `@griffel/react` should go root `package.json` 🤗

Reply:
IMO we should just use eslint ignore.

Also `Menu` stories have used things like react-button for ages, why does it not break there ?

Reply:
> Also Menu stories have used things like react-button for ages, why does it not break there ?

Good question, I am not familiar with `import` plugin and don't see any special config in https://github.com/microsoft/fluentui/blob/b1446dead62e61c9a0f6101d6dfc5a2f780403b8/packages/eslint-plugin/src/configs/react.js#L407-L411.

Reply:
After some digging, we use a custom import resolver for packages that are TS aliased. https://github.com/microsoft/fluentui/blob/73e9916afe9ceab6cb30706d399fffb74be0de79/packages/eslint-plugin/src/configs/react.js#L33

Therefore Fluent internal packages don't count as external imports because eslint resolves them to paths not in `node_modules`

Reply:
having griffel in root is what we want to follow single version policy.,while packages have it specified as dependency because of publishing - the outcome is have 1 griffel version installed in the repo.

Reply:
the plugin/rule sometimes works and sometime doesn't TBH, we should revisit.

Reply:
> After some digging, we use a custom import resolver for packages that are TS aliased.
> 
> https://github.com/microsoft/fluentui/blob/73e9916afe9ceab6cb30706d399fffb74be0de79/packages/eslint-plugin/src/configs/react.js#L33
> 
> Therefore Fluent internal packages don't count as external imports because eslint resolves them to paths not in `node_modules`
> 
> This case could potentially be some edge cases that might just be worth using and eslint ignore instead of adding griffl/react as a devdep WDYT ?

honestly this plugin/config is a bit confusing and made me scratch my head mutliple times...